code-reviews/recipes/Bash.md

```diff
@@ -4,7 +4,7 @@
 
 [CSE](../../CSE.md) developers follow [Google's Bash Style Guide](https://google.github.io/styleguide/shell.xml).
 
-## Linter
+## Linting
 
 [CSE](../../CSE.md) projects must check Bash code with [shellcheck](https://github.com/koalaman/shellcheck) as part of the [CI process](../../continuous-integration/readme.md).
```

Comment:
I think that the Linting sections should be consistent. the other languages state which linters are good and if they have specific installation instructions, how to install.
In this version, the document states it must be part of CI, which seems to be off topic.

On the other hand, I checked the CI document and it does not mention Linting... maybe it should be added there.

Reply:
We just had a discussion in another PR on if we should call this section Linting or Linters - and the consensus was Linting so this is just a change to conform to that